base/server/python/pki/server/cli/ca.py

```diff
@@ -822,6 +823,284 @@ def execute(self, argv):
+                self.print_help()
+                sys.exit()
+
+            else:
+                logger.error('Invalid option: %s', o)
+                self.print_help()
+                sys.exit(1)
+
+        instance = pki.server.instance.PKIServerFactory.create(instance_name)
+        if not instance.exists():
+            logger.error('Invalid instance: %s', instance_name)
+            sys.exit(1)
+
+        instance.load()
+
+        subsystem = instance.get_subsystem('ca')
+        if not subsystem:
+            logger.error('No CA subsystem in instance %s', instance_name)
+            sys.exit(1)
+
+        config = subsystem.get_crl_config()
+        CACRLCLI.print_crl_config(config)
+
+
+class CACRLIPCLI(pki.cli.CLI):
+
+    def __init__(self):
+        super().__init__('ip', 'CA CRL issuing point configuration management commands')
+
+        self.add_module(CACRLIPFindCLI())
+        self.add_module(CACRLIPShowCLI())
+
+    @staticmethod
+    def print_crl_issuing_point_config(id, config, details=False):
+
+        print('  ID: %s' % id)
+        print('  Description: %s' % config.get('description'))
+        print('  Class: %s' % config.get('class'))
+        print('  Enable: %s' % config.get('enable'))
+
+        if not details:
+            return
+
+        print('  Allow Extensions: %s' % config.get('allowExtensions'))
+        print('  Always Update: %s' % config.get('alwaysUpdate'))
+        print('  Auto Update Interval: %s' % config.get('autoUpdateInterval'))
+        print('  CA Certs Only: %s' % config.get('caCertsOnly'))
+        print('  Cache Update Interval: %s' % config.get('cacheUpdateInterval'))
+        print('  Unexpected Exception Wait Time: %s' % config.get('unexpectedExceptionWaitTime'))
+        print('  Unexpected Exception Loop Max: %s' % config.get('unexpectedExceptionLoopMax'))
+        print('  Daily Updates: %s' % config.get('dailyUpdates'))
+        print('  Enable CRL Cache: %s' % config.get('enableCRLCache'))
+        print('  Enable CRL Updates: %s' % config.get('enableCRLUpdates'))
+        print('  Enable Cache Testing: %s' % config.get('enableCacheTesting'))
+        print('  Enable Cache Recovery: %s' % config.get('enableCacheRecovery'))
+        print('  Enable Daily Updates: %s' % config.get('enableDailyUpdates'))
+        print('  Enable Update Interval: %s' % config.get('enableUpdateInterval'))
+        print('  Extended Next Update: %s' % config.get('extendedNextUpdate'))
+        print('  Include Expired Certs: %s' % config.get('includeExpiredCerts'))
+        print('  Min Update Interval: %s' % config.get('minUpdateInterval'))
+        print('  Next Update Grace Period: %s' % config.get('nextUpdateGracePeriod'))
+        print('  Publish On Start: %s' % config.get('publishOnStart'))
+        print('  Save Memory: %s' % config.get('saveMemory'))
+        print('  Signing Algorithm: %s' % config.get('signingAlgorithm'))
+        print('  Update Schema: %s' % config.get('updateSchema'))
```

Comment:
This could be tidied up a bit with a multi-line f-string, but this is minor
```
        print(f"""  Allow Extensions: {config.get('allowExtensions')}
                    Always Update: {config.get('alwaysUpdate')}
        ...
        """)
```

Reply:
Looks like I'll need to use `textwrap.dedent()` and `indent()`. Let me try something..

Reply:
OK, patch updated. It does look cleaner. :+1: